README.md

```diff
@@ -2,3 +2,14 @@
 
 # sermon-helper-py
 Helper files for managing sermon files
+
+
+
+# Usage: 
+(BEFORE USING, MAKE SURE TO INSTALL EYED3 PACKAGE IN PYTHON WITH PIP)- Ask Mark how to do this.
```

Comment:
Thinking we should add "you shouldn't need to do this, but if you do, it's `pip install eyeD3`